KenticoOnboardingApplication/src/KenticoOnboardingApplication.Contracts/Repositories/IConnectionString.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace KenticoOnboardingApplication.Contracts.Repositories
+{
+    public interface IConnectionString
```

Comment:
I guess I would expect the plural here, but it is just a nitpicking detail since there really is just one connection string at this time 😆 
```suggestion
    public interface IConnectionStrings
```